### PR TITLE
fix(whisper): remove workaround INVALID_TOKEN and add missing feature

### PR DESCRIPTION
**Pull request by rebel-eunji**
### 🚀 Summary of Changes

1. Fix workaround INVALID_TOKEN
    a. For now, we've used INVALID_TOKEN to identify the 1st step of decoder.
    b. But it triggers an error related to the shape of logits is [batch_size, vocab_size + 100])
    c. So we removed the workaround by running 1st decoder step in prefill step.
    d. For that, optimum-rbln always allocates one more block to padding the block table.
2. Allow forced_decoder_ids
    a. Whisper model has a feature to instruct the task (e.g. transcribe, translate.. ) and language (e.g. fr, en.. )-- `forced_decoder_ids`
    b. From now on, it is enabled by running the tokens one by one.
3. make script clearer
    a.  Currently, run_whisper.py code frequently causes Python GIL error by importing load_dataset globally . So I've made import load_dataset in the func module.
  

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to https://github.com/RBLN-SW/optimum-rbln/pull/560

---

### ✅ Type of Change

* [ ] 🚀 Release (`release`)
* [x] ✨ Feature (`feature`)
* [x] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [x] 🛠 Bug fix (`fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
5. Verify output: `...`
6. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

**Comment by rebel-eunji** [comment]
@rebel-seinpark I've fixed whisper script in this PR.

**Review by rebel-seinpark — APPROVED** [review]

**Comment by rebel-eunji** [comment]
The fix on `optimum-rbln` will be handled by @rebel-thkim. This PR will be updated to align with that fix once it lands.

File: pyproject.toml [hunk 1]
```diff
@@ -42,7 +42,6 @@ dependencies = [
     "torch",
     "torchvision",
     "torchaudio",
```
Comment (rebel-eunji) [inline pyproject.toml:44]:
```suggestion
    "torchaudio",
    "optimum-rbln>=0.10.4a0",
```

File: vllm_rbln/model_executor/models/optimum/whisper.py [hunk 2]
```diff
@@ -94,58 +90,83 @@ def forward(self, model_input: ModelInputForRBLN, **kwargs) -> torch.Tensor:
...
+            token_sequence = input_ids[0].tolist()
+            step_decoder_input_ids = torch.zeros(self.batch_size, 1, dtype=torch.long)
+            # The decoder runtime is compiled at self.batch_size, so prefill
+            # must still feed every slot. Point unused slots at a scratch
+            # block so their K/V writes don't touch the active prefill block
+            # or any other request's KV cache.
+            assert (
+                model_input.dummy_block is not None
+                and model_input.dummy_block == self.batch_size
+            ), "Whisper prefill requires dummy_block from the scheduler."
```
Comment (rebel-jonghewk) [inline vllm_rbln/model_executor/models/optimum/whisper.py]: can you add a user-friendly error for this? I think this might be the case where the user is using previous optimum version.

Comment (rebel-jonghewk) [reply vllm_rbln/model_executor/models/optimum/whisper.py]: mention that if the user has precompiled artifact, it should be recompiled.

**Review by rebel-jonghewk — APPROVED** [review]
after updating the example code and the error msg. lgtm.
